### PR TITLE
Prepare 1.10 release (WIP)

### PR DESCRIPTION
**Pull request by jgonggrijp**

### CODE REVIEW

**Comment by jgonggrijp** [comment]
@jashkenas I performed all updates to the documentation I could think of, including some changes I didn't contribute myself. I checked the git log for all changes since 1.9.2 and I'm quite sure I completely represented them, at least in the changelog; this is reflected in the five commits above that mention a pull request number.

Other than that, I'm not 100% sure that I did what I'm supposed to do. If I failed to follow a convention somewhere, I apologize. I also performed some backward corrections (I hope); sorry if that was unwarranted. I made small commits so it is easy to see what I did and to change, reorder or omit individual changes.

On the presumption that you have a workflow where you do something like `npm version --minor ; npm publish`, I abstained from the following steps:

- [ ] `npm run build`
- [ ] `npm run doc`
- [ ] update the version number in the `package.json` and the `package-lock.json`

I'll be interested to hear what you think!

File: index.html [hunk 1]
```diff
@@ -476,7 +479,13 @@ <h2>Installation</h2>
         <b>Bower</b> <tt>bower install underscore</tt>
       </li>
       <li>
-        <b>Component</b> <tt>component install jashkenas/underscore</tt>
+        <b>ExtendScript</b> <tt>#include "underscore.js"</tt>
+      </li>
+      <li>
+        <b>Rollup</b> If you want to enable treeshaking and you don&rsquo;t
```
Comment (jashkenas) [inline index.html]: Is this just for Rollup, or for any ESM treeshaking bundler? (i.e. will these instructions work for Webpack too?)

**Comment by jashkenas** [comment]
Okay — v.1.10.1 is out! (I had to bump a tiny patch to fix the relative module references to include file extensions for browser compatibility).

A million thank yous to @jgonggrijp for persevering with this set of changes. Like I said at the beginning — it’s rare to find someone who is actually willing to go the extra mile to make it happen. 👏👏👏

File: index.html (hunk 1 above)
Comment (jgonggrijp) [reply index.html]: To be honest, I don't know. I try to avoid Webpack at all cost.

**Comment by jgonggrijp** [comment]
Hooray! It was a pleasure and I will do this kind of thing again.

Good catch about the module references, I completely forgot about `<script type=module>`.

So this episode is over but I am still a collaborator. In what other ways (if any) would you like me to use the push access? Is it just for the next time I make a large-ish contribution, or would you also appreciate it if, for example, I close an obsolete issue when I find one (such as #1992)? Or maybe even merge a pull request if I'm very sure it is correct?

Also, do you still use Gitter?

**Comment by jgonggrijp** [comment]
Oh by the way, you still have to push the 1.10.1 tag. :-)

**Comment by jashkenas** [comment]
> Oh by the way, you still have to push the 1.10.1 tag. :-)

Thanks! Pushed.

> Also, do you still use Gitter?

I'm afraid I haven't looked at that in years.

> So this episode is over but I am still a collaborator. In what other ways (if any) would you like me to use the push access? Is it just for the next time I make a large-ish contribution, or would you also appreciate it if, for example, I close an obsolete issue when I find one (such as #1992)? Or maybe even merge a pull request if I'm very sure it is correct?

Use it with care and discretion — reaching the bar for collaborator access implies a level of trust that I think you’ve earned by your comportment here. If you want to close an obsolete issue or two, or merge good pull requests, or fix things, I'd very much appreciate it.

**Pull request merged**